test/fhirtypes/ElementDefinition.slicing.test.ts

```diff
@@ -363,9 +363,7 @@ describe('ElementDefinition', () => {
       component.sliceIt('pattern', 'code', false, 'open');
       const systolicBP = component.addSlice('SystolicBP');
       const diastolicBP = component.addSlice('DiastolicBP');
-      systolicBP.sliceIt('pattern', 'interpretation', false, 'open');
```

Comment:
To keep the use case somewhat realistic, maybe instead of removing this (and the other one below it), we should modify it to call `sliceIt` on `component` (which just appends the discriminator to the existing slicing). The test works either way, but in the real world, the re-slice only works if there is another discriminator.